index.js

```diff
@@ -7,6 +7,7 @@ const { Vec3 } = require('vec3')
 
 const sponge = require('./lib/spongeSchematic')
 const mcedit = require('./lib/mceditSchematic')
+const { Block } = require('prismarine-block') // eslint-disable-line
```

Comment:
The linter is right to warn about this line, if its not used it shouldn't be added.

Reply:
this is used to provide typings, not quite sure how I could add the block type to jsdoc without it

Reply:
I'm not sure what typing its adding, in addition of being unnecessary, this line is also wrong: prismarine-block expose a loader function which cannot be de-structured into Block. (There is no way to get a Block class without giving a version)

The documentation shouldn't affect the code written, it should serve it, not the other way arround. By adding an include, even if unused, you modify the code as the include could have side effects.

I don't get why you say you need this, JSDoc is just a commentary, you could put whatever you like in it and it shouldn't complain. And if it works like this, you should really ask yourself why, because this Block class you are importing doesn't exist.

Reply:
replaced with any